apps/site-app/src/containers/App/index.tsx

```diff
@@ -28,24 +27,12 @@ import '../../semantic/semantic.css';
 //import '@thecointech/site-semantic-theme/semantic.less';
 import styles from './styles.module.less';
 
-let hasRun = false;
-
 export const App = () => {
   useFxRatesStore();
   useSidebar();
   const location = useLocation();
   const contextRef = createRef<HTMLDivElement>();
 
-  if (process.env.CONFIG_NAME === 'devlive') {
```

Comment:
Our code shouldn't contain configuration switches like this.

By mocking accounts, we put all of our configuration switching in one place